content/docs/1.8.0/v2-data-engine/features/_index.md

```diff
@@ -15,6 +15,7 @@ aliases:
 - [Selective V2 Data Engine activation](./selective-v2-data-engine-activation)
 - [Filesystem Trim](../../nodes-and-volumes/volumes/trim-filesystem)
 - [Backing Image](../../advanced-resources/backing-image/backing-image)
+- [Volume Encryption](../../advanced-resources/security/volume-encryption)
```

Comment:
Does anything else need to be updated for the volume encryption page, or is it already fully compatible with v1, so nothing needs to be updated?

Reply:
It is fully compatible with v1, all we do is add a suffix for the dm-encrypted device.

Reply:
Should we update the page for the different part?

Reply:
Updated the `volume-encryption` page for the v2 volume. Thanks.